### PR TITLE
[YUNIKORN-2096] Update e2e README

### PR DESCRIPTION
**Pull request by doupache**
### What is this PR for?
Add how to run a specified e2e test in the e2e README

### What type of PR is it?
* [x] - Improvement

### What is the Jira issue?
[YUNIKORN-2096](https://issues.apache.org/jira/browse/YUNIKORN-2096)

### CODE REVIEW

**Comment by doupache** [comment]
@wusamzong @FrankYang0529 @pbacsko  please take a look 🙏

File: test/e2e/README.md [hunk 1]
```diff
@@ -46,13 +46,31 @@ OR follow this doc for deploying go https://golang.org/doc/install
...
+e.g. test/e2e/user_group_limit/
+```shell 
+$ cd test/e2e/
+$ ginkgo -r user_group_limit -v -- -yk-namespace "yunikorn" -kube-config "$HOME/.kube/config"
+```
+
+* Launching specified test.
+e.g. The first augument of ginkgo.It()
+
+ginkgo.It( "Verify_maxapplications_with_a_specific_group_limit" , ... )
```
Comment (chenyulin0719) [inline test/e2e/README.md:70]: image

The format seems broken, would it be better to just use:
* Launching specified test. e.g. Run test with spec name "Verify_maxapplications_with_a_specific_group_limit"

Comment (doupache) [reply test/e2e/README.md:70]: Thanks @chenyulin0719  for reviewing. fair point 👍

**Review by wilfred-s — APPROVED** [review]
LGTM, @manirajv06 a second review please

**Review by pbacsko — APPROVED** [review]
+1 LGTM
checked the commends in the console

**Pull request closed without merging**